lib/TelemetryFeed.jsm

```diff
@@ -641,9 +634,9 @@ this.TelemetryFeed = class TelemetryFeed {
   }
 
   sendEvent(event_object) {
```

Comment:
So if I got this, the events we're losing are `countPinnedTab`, `handleUndesiredEvent`, `sendPageTakeoverData` and `TELEMETRY_PERFORMANCE_EVENT`.

We're keeping because they are already migrated are: `handleUserEvent` `handleImpressionStats` `sendDiscoveryStreamLoadedContent` `sendDiscoveryStreamImpressions` `endSession`. I don't see where you remove the `sendEvent` call in `endSession` and `handleUserEvent`, could we, or is there a reason to keep it around for now?

Reply:
Actually, `sendEvent` takes care of all pings routed to Tiles. So this patch will make us lose all those pings.

Like I mentioned earlier, we're going to add them back one by one in the following releases. As planned, the event pings will be done first in 72.

Reply:
Hm, I think I'm still missing something.

It looks like to me that `handleUserEvent` and `endSession` have already been migrated, and thus can have the `sendEvent` removed like you do in `handleImpressionStats` `sendDiscoveryStreamLoadedContent`, etc, or am I missing something?

Reply:
We're good because `sendUTEvent` is not the same as the stuff we do in handleImpressionStats sendDiscoveryStreamLoadedContent, etc